.github/workflows/quick_deploy.yml

```diff
@@ -0,0 +1,21 @@
+
+name: Quick deploy trough code push
```

Comment:
1. rename this file to codepush.yml
2. rename flow to iOS & Android Quick Release